### ISSUE TITLE
[PM-1921] arm arch string not consistent on linux and mac platforms

### ISSUE THREAD

**Issue by mayani** [pegasus-isi/pegasus#2034]
A LIGO user reported the following error on support. 
I encountered an error while using Pegasus. Here are the details:
(research) neereshkumarperla@Neereshs-MacBook-Pro PegasusExample % pegasus-plan -Dpegasus.catalog.site.file=sites.xml \
        -Dpegasus.catalog.transformation.file=tc.txt \
        -Dpegasus.catalog.replica=File \
        -Dpegasus.catalog.replica.file=rc.txt \
        -Dpegasus.register=false \
        -s local -o local --dir submit \
        --submit DAXFILE.dax
2023.07.19 18:37:43.219 EDT: [FATAL ERROR]
 [1] java.lang.RuntimeException: Unable to determine architecture from arm64_macos_13 at edu.isi.pegasus.common.util.Version.getArchitecture(Version.java:136)
ERROR while logging metrics The metrics file location is not yet initialized
I have attached a screenshot for the reference.
Please let me know how can

**Reporter:** @vahi
**Resolution:** Fixed
**Watchers:**
@papajim
@mayani
@vahi

**Comment by mayani** [comment]
**Author:** @vahi

as part of #2024 we added aarch64 as arch type for arm64 platforms. that is what uname -a returns on LINUX
however on MAC uname -a returns arm64. 
after discussing, we decided to pick one as the enum value. since condor also picks up aarch64 for their arm builds. we will keep it consistent, and update our build system (getsystem script that calls out to pegasus-lite-common) to set arch to aarch64 when arm64 is detected
